### PR TITLE
Draft slides for Exercise 2

### PR DESCRIPTION
**Pull request by cpacey**

### CODE REVIEW

File: docs/slides.md [hunk 1]
```diff
@@ -29,7 +29,48 @@ Carl Pacey and Mark Tse
...
+# Exercise 2: Specification by Example
+
+```http
+GET /users/1045
+```
+
+```json
+{
+  "id": 1045,
+  "userName" = "cpacey",
```
Comment (cpacey) [inline docs/slides.md]: Ha!  That's not right at all.

File: docs/slides.md [hunk 2]
```diff
@@ -29,7 +29,48 @@ Carl Pacey and Mark Tse
...
+  "userName": "cpacey",
+  "isActive": true,
+  "buzzWord": "isawesome"
+}
+```
+
+| `GET` |      Result      |
+|:----------|:-------------|
+| `/fizzbuzz/14/users/1045` |  `29` |
+| `/fizzbuzz/30/users/1045` | `cpacey` |
```
Comment (neverendingqs) [inline docs/slides.md]: 30 is divisible by 5, and technically should return `cpaceyisawesome`.

File: docs/slides.md [hunk 3]
```diff
@@ -29,7 +29,48 @@ Carl Pacey and Mark Tse
...
+| `/fizzbuzz/50/users/1045` | `isawesome` |
+| `/fizzbuzz/75/users/1045` | `cpaceyisawesome` |
```
Comment (neverendingqs) [inline docs/slides.md]: Do we need to explain what FizzBuzz is?

**Review by neverendingqs — CHANGES_REQUESTED** [review]

File: docs/slides.md (hunk 2 above)
Comment (cpacey) [reply docs/slides.md]: Wow I suck.

File: docs/slides.md (hunk 3 above)
Comment (cpacey) [reply docs/slides.md]: Good call.

Comment (cpacey) [reply docs/slides.md]: I've added a slide for this, but I'm going to punt on the content for the moment.

**Review by neverendingqs — APPROVED** [review]

**Pull request merged**